src/main/java/com/vinyl/model/Token.java

```diff
@@ -0,0 +1,55 @@
+package com.vinyl.model;
+
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.OneToOne;
+import javax.validation.constraints.NotNull;
+
+import org.hibernate.annotations.GenericGenerator;
+
+
+@Entity
+public class Token {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO, generator = "native")
+	@GenericGenerator(name = "native", strategy = "native")
+	private Integer id;
+	
+	@Column(name="HASH")
+	@NotNull
+	private String hash;
+
+	@OneToOne
+	@NotNull
+	@JoinColumn(name="user_id")
+	private User user;
```

Comment:
add a validUntil field, token should not be valid forever, make it valid maybe for 1 month after it is created

Reply:
done